libs/shared/ui-elements/src/lib/prio-button/prio-button.component.ts

```diff
@@ -9,10 +11,20 @@ import { Component, OnInit, Input, Output, EventEmitter, HostBinding } from '@an
 
 export class PrioButtonComponent implements OnInit {
 
-  inputFocus = false;
+  @Input() style = 'solid' ;  //solid || outline || flat
+  @Input() type = 'btn-primary' ;  //btn-primary || btn-secondary || btn-danger, ...etc
+  @Input() size = 'normal' ;   //normal || small
+  @Input() isBlock = false ;  // if true: button take dislay as block
+  @Input() isIcon = false ;  // if true: turns button view to icon (also if no text exists) and it takes class btn-icon
+  @Input() label = '' ;    //buttontext
```

Comment:
We should delete this input it's not used

Reply:
I do not get why these are not used
so if isBlock is true, it will have the class btn-block
same for  isIcon  btn-icon
and size= small   btn-sm
?
@Abdallah-khalil

Reply:
sorry this comment is only for line 19